### PR TITLE
TSE review

### PR DESCRIPTION
**Pull request by nikitin2009**

### CODE REVIEW

**Comment by rvvergara** [comment]
Hi Alex and Ceda,

I just have a few suggested revisions:

1. It would be great for your README to have a short description about the project (i.e, any special feature [like responsiveness] that you like to showcase). Also instead of writing your respective usernames in the README, first names would be better along with links to your respective github profiles. For guidance about markdown - https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet.

2. One more thing about the README, it would be so cool if you could have a demo site (you can use raw.githack.com) so anyone who visits your project will be able to see your finished product (and this is great for your portfolios). 

3. Your index.html passed the html validator tool. That's great! I went through the code here and all is well organized (indentation and all). Great job guys!

4. Minor suggested changes for your style.css. Some styles (like the display: flex which appeared 11 lines in total) would be better written as a single style. Like this for instance:

`.single-deal, .main-header nav, .top-menu, ...etc { display: flex}`

This way it will be easier for you to make changes to those elements that have the exact same feature (in this case, the display: flex style). That also helps you later on when you want to review your code as it'll be more readable and easier for your to identify specific styling.

5. Responsiveness is the main feature we want for this project, and you just rock! :) Just a few things I would like to suggest as improvements:

a. The images are too fluid. They tend to appear warped at certain browser widths. Not a deal breaker here but would be great if you could fix that.

b. While it's responsiveness we are after here, it would be great to also improve the type faces, the paddings (for instance the left padding in the GEAR section at the first row of the web page). In short, let's just add a little bit more into the presentation.

6. Lastly, great work in your git flow. You commit often and with each feature you change. This is good practice and keep it up :) 

Other than the above changes, you did a great job. After you edited your project based on the above you may again submit another code review.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>TNW</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+  <header>
```
Comment (heyapricot) [inline index.html:27]: This looks nothing like the original when the width is ~764px

File: index.html [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+          <li><a href="#">Investing 2.0</a></li>
+        </ul>
+        <button class="search">Search</button>
+      </nav>
+    </div>
+  </header>
+
+  <section class="top-section">
+
+    <div class="main-article">
```
Comment (heyapricot) [inline index.html:62]: Where are the pictures? Also, no fonts or style for the text?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,515 @@
...
+        </div>
+      </div>
+
+    </div>
+
+  </section>
+
+
+
+    <section class="latest-news">
```
Comment (heyapricot) [inline index.html:106]: Header and spacing don't match the original

File: index.html [hunk 4]
```diff
@@ -0,0 +1,515 @@
...
+              <h4>Forza axes emotes after Fortnite controversy</h4>
+              <span>Rachel Kaser &#8226; 12 hours ago</span>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="index-co">
```
Comment (heyapricot) [inline index.html:181]: Review the alignment and match the appearance of the original

File: index.html [hunk 5]
```diff
@@ -0,0 +1,515 @@
...
+          <span>Driivz grabs $12 mln<span class="pe-hub"> on PE Hub</span></span>
+
+        </div>
+
+      </div>
+    </div>
+
+    </section>
+
+    <section class="categories">
```
Comment (heyapricot) [inline index.html:266]: Match the appearance of the original

File: index.html [hunk 6]
```diff
@@ -0,0 +1,515 @@
...
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+          </ul>
+        </div>
+
+      </section>
+
+    <section class="latest-deals">
```
Comment (heyapricot) [inline index.html:532]: Again, match the appearance of the original. Pictures should be squares. Use the same colors. Check the font size and style. Also, the spaces between elements.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,515 @@
...
+            <h4>Apple launches battery cases for the iPhone XS, XS Max, and XR at $129</h4>
+            <span>Abhimanyu Ghoshal &#8226; 6 hours ago</span>
+          </div>
+        </div>
+      </div>
+    </section>
+
+
+
+  <footer>
```
Comment (heyapricot) [inline index.html:597]: Where are the icons? The font style is also different.

**Review by heyapricot — CHANGES_REQUESTED** [review]
# Good first attempt. Needs a bit more of work
Hello! 👋🏼 You are on the right path, but the project looks different enough from the original page that I have to request for you to invest more time in this project. Please address the comments, and after that submit a new code review request. Thanks! 

Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

File: index.html (hunk 1 above)
Comment (nikitin2009) [reply index.html:27]: @heyalvaro sorry I don't quite understand your comments. Not sure what you meant by pointing to the code snippets. Did you mean the "header" element in this case? So, you are pointing at elements?
Also, does this "~764px" mean "approximately" 764px?

File: style.css [hunk 8]
```diff
@@ -0,0 +1,717 @@
...
+  color: #333;
+}
+
+.index-wrap {
+  width: 100%;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.single-index {
```
Comment (heyapricot) [inline style.css]: `Duplicate selector`. You first used this selector on line 286

**Review by heyapricot — CHANGES_REQUESTED** [review]
# Almost there 🧗🏻‍♂️

👋🏼 Hello! The page looks so much better. Still, I have to request solving some minor issues before I approve. Please fix the issues listed below and then submit a new code review request. Thanks!

- When the width is approximately 764px: 

1. The style of **ALL** headers should match the original
screen shot 2019-02-05 at 11 01 33

2. The navigation links are arranged in a different way 
screen shot 2019-02-05 at 11 02 21



3. The Hard Fork Section (And all the similar ones) show bigger pictures and the pictures are usually squares. The pictures should match the size of the originals
screen shot 2019-02-05 at 11 03 13

4. On the Latest Deals section, the style of the header doesn't match and the pictures are not squares
screen shot 2019-02-05 at 11 05 43

- When the width of the page is approximately > 764px

1. There is no logo
2. The navbar links don't have a style that matches the original

screen shot 2019-02-05 at 11 10 57

**Comment by heyapricot** [comment]
# You did it 🏅

You can merge! 
One more comment: There is no logo when the page has full width. Please address that and after that, you are ready to move on. No need from my side for further review.	
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

**Review by heyapricot — APPROVED** [review]

**Pull request merged**